### DISCUSSION TITLE
[need help] Flicker problem in myself react-fabric

### DISCUSSION THREAD

**Discussion by vaynevayne** [fabricjs/fabric.js#10240] [Q&A]
https://stackblitz.com/edit/vitejs-vite-eyhjps?file=src%2FReactFabric%2Fcomponents%2FCanvas%2Findex.tsx

When resize, will flicker, this is not fabricjs problem, if free, please help me to take a look at it

**Comment by asturur** [comment]
@vaynevayne and in general other people, this is not the way to open a discussion or an issue either.
Throwing a link that  has a complex application ( 2 minutes is not build yet )and not even spending a word about what is flickering.

This is wasting people's time.
